img_proof/ipa_cloud.py

```diff
@@ -101,7 +102,8 @@ def __init__(
         enable_secure_boot=None,
         enable_uefi=None,
         log_callback=None,
-        prefix_name=None
+        prefix_name=None,
+        retry_count=None
```

Comment:
Should this be `0` ?

Reply:
No everything is None because the dictionaries are chain mapped. We have to filter out None values because chain map works on keys as opposed to truthyness. https://github.com/SUSE-Enceladus/img-proof/blob/2b19ce6e4849e94c324ff9f4606b810bdd5987d1/img_proof/ipa_cloud.py#L111 There's a default dictionary https://github.com/SUSE-Enceladus/img-proof/blob/2b19ce6e4849e94c324ff9f4606b810bdd5987d1/img_proof/ipa_cloud.py#L55 that's mapped as the final set of values for anything that has a default.